src/routers/index.ts

```diff
@@ -23,8 +23,8 @@ rootRouter.use('/bulkUploads', bulkUploadsRouter);
 rootRouter.use('/dataProviders', dataProvidersRouter);
 rootRouter.use('/funders', fundersRouter);
 rootRouter.use('/opportunities', opportunitiesRouter);
-rootRouter.use('/organizations', organizationsRouter);
-rootRouter.use('/organizationProposals', organizationProposalsRouter);
+rootRouter.use('/changemakers', changemakersRouter);
```

Comment:
Perhaps these should be above `/dataProviders` to have alphabetical order in the file.